src/Umbraco.Cms.Integrations.Automation.Zapier/Controllers/PollingController.cs

```diff
@@ -39,12 +50,16 @@ public PollingController(IContentService contentService, IUserValidationService
             Options = new ZapierSettings(ConfigurationManager.AppSettings);
 #endif
 
+            _umbracoContextFactory = umbracoContextFactory;
+
             _contentService = contentService;
 
+            _zapierFormService = zapierFormService;
+
             _userValidationService = userValidationService;
         }
 
-        public List<Dictionary<string, string>> GetContent()
+        public IEnumerable<PublishedContentDto> GetSampleContent()
```

Comment:
I'm a bit confused about this controller.  `GetSampleContent` and `GetSampleForm` don't seem to be consistent.  By that I mean the former is returning a partial definition of some published content, but the latter all the forms.  Seems to be if anything it should be exposing form submissions, not the form definitions (i.e. what would be the content of the zap information).

And `GetContentByAlias` is a concern as it looks like it's getting all the published content on the site for a given content type, which could be a lot and I don't think we'd want to respond to that on a poll.

Reply:
Updated.